stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/proto/definition/File.java

```diff
@@ -80,7 +80,7 @@ public String getFileDefinition() {
         fileDefinition.append("syntax = \"").append(fileDescriptorProto.getSyntax()).append("\";").append
                 (ServiceProtoConstants.NEW_LINE_CHARACTER);
         if (!"".equals(fileDescriptorProto.getPackage())) {
-            fileDefinition.append("module ").append(fileDescriptorProto.getPackage()).append(";").append
+            fileDefinition.append("package ").append(fileDescriptorProto.getPackage()).append(";").append
                     (ServiceProtoConstants.NEW_LINE_CHARACTER);
```

Comment:
Why have we changed 'module' to 'package'?

Reply:
@NatashaWso2 this is not ballerina package. This is proto file package. We are generating proto file at compile time and ballerina module needs to map to protobuf package